connexion/security/__init__.py

```diff
@@ -4,6 +4,23 @@
 from .async_security_handler_factory import AbstractAsyncSecurityHandlerFactory  # NOQA
 from .security_handler_factory import AbstractSecurityHandlerFactory  # NOQA
 
+
+def not_installed_error(exc):  # pragma: no cover
```

Comment:
We have the same function in `connexion.__init__.py`.
Can we move it to `utils.py`?

Reply:
Good point, done!